.github/workflows/security-trivy-scan-callable.yml

```diff
@@ -0,0 +1,123 @@
+name: Security - Scan Docker Image With Trivy
+
+on:
+  workflow_dispatch:
+    inputs:
+      image_ref:
+        description: 'Full image reference to scan e.g ghcr.io/n8n-io/n8n:latest'
+        required: true
+        default: 'ghcr.io/n8n-io/n8n:latest'
+  workflow_call:
+    inputs:
+      image_ref:
+        type: string
+        description: 'Full image reference to scan e.g ghcr.io/n8n-io/n8n:latest'
+        required: true
+    secrets:
+      SLACK_WEBHOOK_URL:
+        required: true
+
+jobs:
+  security_scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+
+      - name: Pull Docker image
+        run: |
+          docker pull ${{ inputs.image_ref }}
```

Comment:
this can now be used to scan any arbitrary image from **any** registry, which I think has a potential for abuse.
are we certain that this is what we want to do?
If you say so, I'll approve.